src/app/zap/gen_config.zapt

```diff
@@ -0,0 +1,25 @@
+{{chip_header}}
+
+// Enclosing macro to prevent multiple inclusion
+#ifndef __ZAP_GEN_CONFIG__
+#define __ZAP_GEN_CONFIG__
+
+// User options for plugin Binding Table Library
+#define EMBER_BINDING_TABLE_SIZE 10
+
+/**** Network Section ****/
+#define EMBER_SUPPORTED_NETWORKS (1)
+
+
+#define EMBER_APS_UNICAST_MESSAGE_COUNT 10
+
+/**** CLI Section ****/
+#define EMBER_AF_GENERATE_CLI
```

Comment:
We are getting rid of the files that uses this macro in #3688 (and this code was not used anyway). So we can probably get rid of this macro here as well.